packages/ckeditor5-engine/tests/view/filler.js

```diff
@@ -119,4 +120,12 @@ describe( 'filler', () => {
 			document.body.removeChild( iframe );
 		} );
 	} );
+
+	describe( 'MARKED_NBSP_FILLER', () => {
+		it( 'should return node with correct HTML', () => {
+			const node = MARKED_NBSP_FILLER( document ); // eslint-disable-line new-cap
+
+			expect( node.outerHTML ).to.equal( '<span data-cke-filler="true">&nbsp;</span>' );
+		} );
+	} );
```

Comment:
Unfortunately, this test is not verifying whether `innerText` is used. It's passing even without changing the helper implementation. It would be good to have it tested to ensure the implementation won't get rolled back.

Reply:
@niegowski , I added one more test case, can you pls take another look? Thank you.